README.md

```diff
@@ -91,6 +91,41 @@ render(
 
 > ⚡️ Check out the [live and interactive demo](https://esm.codes/#Ly8gT2NlYW53aW5kIGRlbW8gYnkgQGx1a2VqYWNrc29ubgovLyAtLS0tLS0tLS0tLS0tLS0tCiAgICAKaW1wb3J0IHsgcmVuZGVyLCBoIH0gZnJvbSAnaHR0cHM6Ly91bnBrZy5jb20vcHJlYWN0P21vZHVsZSc7CmltcG9ydCBodG0gZnJvbSAnaHR0cHM6Ly91bnBrZy5jb20vaHRtP21vZHVsZSc7CmltcG9ydCBvdyBmcm9tICdodHRwczovL3VucGtnLmNvbS9vY2VhbndpbmQnOwoKY29uc3QgaHRtbCA9IGh0bS5iaW5kKGgpOwoKcmVuZGVyKAogIGh0bWxgCiAgICA8ZGl2IGNsYXNzTmFtZT0ke293YAogICAgICBoLWZ1bGwKICAgICAgYmctcHVycGxlLTUwMAogICAgICBmbGV4CiAgICAgIGl0ZW1zLWNlbnRlcgogICAgICBqdXN0aWZ5LWNlbnRlcgogICAgYH0+CiAgICAgIDxoMSBjbGFzc05hbWU9JHtvd2AKICAgICAgICB0ZXh0LXdoaXRlCiAgICAgICAgZm9udC1ib2xkCiAgICAgICAgZm9udC1zYW5zCiAgICAgICAgaG92ZXI6cm90YXRlLTMKICAgICAgICBob3ZlcjpzY2FsZS0xNTAKICAgICAgICBob3ZlcjpjdXJzb3ItcG9pbnRlcgogICAgICBgfT5IZWxsbyBXb3JsZDwvaDE+CiAgICA8L2Rpdj4KICBgLAogIGRvY3VtZW50LmJvZHkKKTs=)
 
+## Server-side rendering (SSR)
+
+Oceanwind supports SSR through Otion. Consider the following example:
+
+```js
+import { h } from 'preact';
+import render from 'preact-render-to-string';
+import htm from 'htm';
+import {
+  filterOutUnusedRules,
+  getStyleTag,
+  VirtualInjector
+} from "otion/server";
+import { setup, themed } from 'oceanwind';
+
+const injector = VirtualInjector();
+setup({ injector });
+
+const html = htm.bind(h);
+const ow = themed({});
+const style = {
+  main: ow`clearfix`,
+};
+
+const app = html`<main className=${style.main}>hello oceanwind</main>`
+const appHtml = render(app)
+const styleTag = getStyleTag(filterOutUnusedRules(injector, appHtml))
+
+// Inject styleTag to your HTML now. 
+```
+
+Oceanwind exposes `hydrate` from Otion for the client-side.
```

Comment:
We should figure out what's the purpose of `hydrate` in this case. Is it for theming?

Reply:
We should.. I wonder if @JoviDeCroock knows why 🤔 I'll message him and see.

Reply:
Afaik it takes all the rules from the SSR-run and attaches them to their corresponding vnode.

Reply:
Based on https://github.com/kripod/otion/search?q=hydrate&unscoped_q=hydrate my impression you would need to hydrate only in Gatsby/Next setups. The related code comment is "Marks server-rendered CSS identity names as available to avoid re-injecting them to the style sheet during runtime." so that definitely implies it has to do with React rendering.

Since I'm using a static render (no vdom or anything) I don't need it for my use case but likely we should keep the `hydrate` mention there.

Reply:
Ok, well given that it doesn't really affect anything else but might enable Next/Gatsby setups then I'll leave it in for now.